packages/node_modules/@node-red/editor-client/src/js/keymap.json

```diff
@@ -90,6 +90,7 @@
         "alt-a m": "core:align-selection-to-middle",
         "alt-a c": "core:align-selection-to-center",
         "alt-a h": "core:distribute-selection-horizontally",
-        "alt-a v": "core:distribute-selection-vertically"
+        "alt-a v": "core:distribute-selection-vertically",
+        "alt-l": "core:split-wire-with-link-nodes"
```

Comment:
I wonder if this should be a sequence rather than single keystroke. I can imagine having other `split-wire-with-` type actions  (Hello Junctions!) that would suit having a common family of shortcuts.

Does `alt-l l` (`l` for `link`) make sense? We could then have `alt-l j` for split with junction.

Reply:
yes - there are starting to be so many variations of splitting, slicing, rejoining wires I think they need to be grouped in some way so you have a chance of remembering at least some of them and guess the others are "nearby"

Reply:
I agree, some grouping would aid comprehension (so long as they are also in the action list with the shortcuts visible)

I personally prefer <kbd>CTRL-K</kbd> as a chord starter (it is common chord starter) 

And as J and L are adjacent, it wouldnt be unreasonable (fingerwise) to set 
* <kbd>CTRL-K</kbd> <kbd>J</kbd> - Create Junction
* <kbd>CTRL-K</kbd> <kbd>L</kbd> - Create Links

If <kbd>ALT-?something?</kbd> is prefered, then perhaps <kbd>ALT-W</kbd> (W being wires?)